pkg/cmd/util/output/backup_describer.go

```diff
@@ -70,11 +70,9 @@ func DescribeBackup(
 			}
 		}
 
-		if status.Phase == velerov1api.BackupPhasePartiallyFailed {
-			d.Println()
-			d.Printf("Errors:\t%d\n", status.Errors)
-			d.Printf("Warnings:\t%d\n", status.Warnings)
-		}
+		d.Println()
```

Comment:
I like this change. It is a good step towards unifying restore and backup describe output. (cc @sseago)

Reply:
Yeah I tried to be consistent between the two, it's a lot easier for users that way.